src/adservice/src/main/java/hipstershop/AdService.java

```diff
@@ -226,6 +221,12 @@ public static void main(String[] args) throws IOException, InterruptedException
 
     initializeAds();
 
+    // Registers all RPC views.
+    RpcViews.registerAllViews();
+
+    // Registers logging trace exporter.
+    LoggingTraceExporter.register();
```

Comment:
Why did you move this section of code?  I thought it was better to register the views earlier to avoid losing data.

Reply:
I have two exporters, Prometheus and Stackdriver, initializing views  within initStackdriver() makes it look like registeration of views is part of Stackdriver. In fact it is common to both Stackdriver and Prometheus. So I moved it common place in main().

Reply:
Sorry, I misread the code and thought this section was moved farther down in initStackdriver.  This change is how I expected initialization to work.